server/controllers/questions.js

```diff
@@ -18,11 +18,13 @@ questionsRouter.post("/", async (req, res) => {
         const question = await QuestionModel.create(req.body);
         res.status(201).json(question);
     } catch (error) {
-        if (error.name === "ValidationError") {
+        if (error.code === 11000) {
```

Comment:
I did a lil search and found out this was a duplicateKey error

this is a bit nitpicky but there should be a comment about this "magic number" or a variable that signals what 11000 is

Reply:
thanks for the reminder! comment added in the code